pkg/cluster/executor/ssh.go

```diff
@@ -301,7 +301,17 @@ func (e *NativeSSHExecutor) Execute(cmd string, sudo bool, timeout ...time.Durat
 		defer cancel()
 	}
 
-	args := []string{"ssh", "-o", "StrictHostKeyChecking=no"}
+	ssh := "ssh"
+
+	if val := os.Getenv(localdata.EnvNameSSHPath); val != "" {
+		if isExec := utils.IsExecBinary(val); !isExec {
+			return nil, nil, fmt.Errorf("The specified SSH in the environment variable `%s` does not exist or is not executable", localdata.EnvNameSSHPath)
+		}
+		ssh = val
+	}
+
+	args := []string{ssh, "-o", "StrictHostKeyChecking"}
```

Comment:
This is why it report `command-line line 0: missing argument.` I think. should be `StrictHostKeyChecking=no`

Reply:
This option has set before, why there was no failure before. 😿

Reply:
This option should have a value, that's 'no' or 'yes', it can't be omit

So `ssh -o StrictHostKeyChecking=no` is ok but `ssh -o StrictHostKeyChecking` not